.github/workflows/bazel.yml

```diff
@@ -25,6 +25,36 @@ jobs:
               run: |
                   tools/bazelisk build --compilation_mode=fastbuild  //...
 
+    generate-artifacts:
+        name: "Artifacts"
+        runs-on: ubuntu-latest
+        env:
+            SPLINE: pi3hat
+            SPLINE_PATH: ./bazel-out/aarch64-opt/bin/spines/
+        steps:
+            - name: "Checkout sources"
+              uses: actions/checkout@v3
+
+            - name: "Install dependencies"
+              run: |
+                  python -m pip install --upgrade pip
+                  python -m pip install numpy
+                  sudo apt install libncurses5
```

Comment:
We shouldn't need these dependencies to build the spine (neither ncurses nor Python ones, the pi3hat is fully C++).

Reply:
Indeed, no need for the Python dependencies, but libncurses5 is required. 

Get the following error message otherwise : "external/org_llvm_clang/bin/clang: error while loading shared libraries: libtinfo.so.5: cannot open shared object file: No such file or directory"

Solutions are either to install libncurses5, or make symbolic links to libncurses6 (the version installed)

Reply:
Pushed anupdated version with juste the required dependency